Source/WebCore/Modules/fetch/FetchResponse.cpp

```diff
@@ -168,6 +163,21 @@ ExceptionOr<Ref<FetchResponse>> FetchResponse::redirect(ScriptExecutionContext&
     return redirectResponse;
 }
 
+ExceptionOr<Ref<FetchResponse>> FetchResponse::staticJson(ScriptExecutionContext& context, JSC::JSValue data, Init&& init)
+{
+    String jsonString = JSC::JSONStringify(context.globalObject(), data, 0);
+    if (jsonString.isNull())
+        return Exception { TypeError, "Value doesn't have a JSON representation"_s };
+
+    String ignored;
+    auto bodyResult = FetchBody::extract(WTFMove(jsonString), ignored);
```

Comment:
This is a no-op in our implementation, I would think we do not need this since body can be a String.

Reply:
`FetchBody` doesn't have any public constructors, and it seems like the only ways to create an instance are the `extract` and `fromFormData` static methods.